### PR TITLE
docs: initial commit of guides

### PR DESCRIPTION
**Pull request by plisovyi**

### CODE REVIEW

File: .github/workflows/jekyll-gh-pages.yml [hunk 1]
```diff
@@ -35,14 +35,17 @@ jobs:
...
       - run: yarn
       - run: yarn build
+      - run: yarn workspace engine-guides build
       - run: yarn docs build
       - name: Build with Jekyll
         uses: actions/jekyll-build-pages@v1
         with:
           source: ./_docs
           destination: ./_site
+      - run: sudo mkdir -p ./_site/guides && sudo mv ./guides/build/* ./_site/guides
```
Comment (Ari-Kishon) [inline .github/workflows/jekyll-gh-pages.yml:48]: We do these actions in codux ci with no sudo.
unsure why its needed here. could be a permissions issue

**Review by Ari-Kishon — APPROVED** [review]

File: .github/workflows/jekyll-gh-pages.yml (hunk 1 above)
Comment (plisovyi) [reply .github/workflows/jekyll-gh-pages.yml:48]: Codux on AWS, and this runners? Or is it unrelated?

**Review by oleksandraab — APPROVED** [review]

File: yarn.lock [hunk 2]
```diff
@@ -6761,6 +6770,14 @@ react-is@^18.0.0:
   resolved "https://registry.yarnpkg.com/react-is/-/react-is-18.2.0.tgz#199431eeaaa2e09f86427efbb4f1473edb47609b"
   integrity sha512-xWGDIW6x921xtzPkhiULtthJHoJvBbF3q26fzloPCK0hsvxtPVelvftw3zjbHWSkR2km9Z+4uxbDDK/6Zw9B8w==
 
+react@^17.0.2:
+  version "17.0.2"
```
Comment (AviVahl) [inline yarn.lock:6774]: So we now have both react 17 and 18 in this repo?

Comment (plisovyi) [reply yarn.lock:6774]: We had them both for short time between this PR merge and update to Docusaurus 3, where we collapsed both to one version (18)

**Pull request merged**